From: Caius Durling <dev@caius.name>
Subject: Pin the POSIX version for CentOS 7

* Makefile: Pin the POSIX version for CentOS 7.

Origin: upstream, https://github.com/fac/auth-script-openvpn/commit/19eefe089be06f58b788053f702f9498b4243bfa
--- a/Makefile
+++ b/Makefile
@@ -1,5 +1,5 @@
 CC := gcc
-CFLAGS := -std=c99 -Wall -Wextra -Wformat-security -fstack-protector-strong -D_FORTIFY_SOURCE=2
+CFLAGS := -O -std=c99 -Wall -Wextra -Wformat-security -fstack-protector-strong -D_FORTIFY_SOURCE=2 -D_POSIX_C_SOURCE=200809L
 LDFLAGS := -fPIC -shared
 
 all: plugin
